server/runtime/src/main/codestarts/quarkus/openapi-generator-codestart/java/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+quarkus:
```

Comment:
For OpenAPI Generator Server extension, we just need this property to start: `quarkus.openapi.generator.spec=openapi.yml`